src/domains/contact/pages/Contacts/Contacts.tsx

```diff
@@ -40,17 +41,25 @@ const ContactsHeaderExtra = ({ showSearchBar, onSearch, onAddContact }: Contacts
 };
 
 type ContactsProps = {
-	contacts: any[];
-	networks: NetworkData[];
+	contacts?: any[];
+	networks?: NetworkData[];
 	onSearch?: any;
 };
 
 export const Contacts = ({ contacts, networks, onSearch }: ContactsProps) => {
+	const { env, persist } = useEnvironmentContext();
+	const { t } = useTranslation();
+	const activeProfile = useActiveProfile();
+
+	const [profileContacts] = useState<any>(contacts);
 	const [createIsOpen, setCreateIsOpen] = useState(false);
 	const [contactToDelete, setContactToDelete] = useState(null);
-	const activeProfile = useActiveProfile();
 
-	const { t } = useTranslation();
+	const [availableNetworks] = useState<NetworkData[]>(networks || env.availableNetworks());
+
+	useEffect(() => {
+		// setContacts(activeProfile?.contacts().values() || []);
```

Comment:
@goga-m was this forgotten to be uncommented or can it be removed together with the whole `useEffect` since it would be empty after removing that line?

Reply:
@faustbrian I would uncomment it after I finished with Contact listing (rendering data from platform-sdk) that's why I left it commented for now. Maybe better to mark it as // TODO:

Reply:
After our discussion with Eddie, he will continue with the Contact Listing, so will push a commit to clean this up.

Reply:
Mark it as TODO because at the moment it looks like it's just dead code.